src/content/layout/builtin_layout.cc

```diff
@@ -91,22 +91,39 @@ BuiltinLayout::BuiltinLayout(std::shared_ptr<ContentManager> content)
     container["Audio"] = container.at("Audio/audioRoot");
     container["Audio"]->addMetaData(MetadataFields::M_CONTENT_CLASS, UPNP_CLASS_AUDIO_ITEM);
     container["All Audio"] = container.at("Audio/allAudio");
-    container["All Songs"] = container.at("Audio/allSongs");
+    // Not clear if it shall be possible to disable this container...
     chain["/Audio/All Audio"] = this->content->addContainerTree({ container["Audio"], container["All Audio"] }, nullptr);
+
+    container["All Songs"] = container.at("Audio/allSongs");
+
     container["All - full name"] = container.at("Audio/allTracks");
-    chain["/Audio/All - full name"] = this->content->addContainerTree({ container["Audio"], container["All - full name"] }, nullptr);
+    if (blOption->get("Audio/allTracks")->getEnabled()) {
+        chain["/Audio/All - full name"] = this->content->addContainerTree({ container["Audio"], container["All - full name"] }, nullptr);
+    }
```

Comment:
Maybe you can introduce some flag-dict
`hasContainer` to link with chain and use it in `add...`

Reply:
I think there are many things to refactor/improve in the Box configurations... I'm preparing a separate PR which shall clean up many stuff there